packages/plugin-import-export/src/export/getCreateExportCollectionTask.ts

```diff
@@ -41,7 +45,9 @@ export const getCreateCollectionExportTask = (config: Config): TaskHandler<any,
       await createExport({ input, req, user })
 
       return {
-        success: true,
+        output: {
+          success: true,
```

Comment:
Tasks themselves already maintain a successful / failed state - no need to add your own and save it to the task output. I'd just use `return { output: {} }`

Reply:
Oh great point!